releng/docker/image-build.sh

```diff
@@ -1,12 +1,51 @@
 #!/bin/sh
```

Comment:
This is unrelated to the PR, but why are we using `/bin/sh` in this script, with `/bin/bash` used virtually everywhere else?